app/src/main/java/com/google/android/ground/MainActivity.kt

```diff
@@ -217,6 +217,11 @@ class MainActivity : AbstractActivity() {
   }
 
   private fun navigateTo(directions: NavDirections) {
-    navHostFragment.navController.navigate(directions)
+    navHostFragment.navController.currentDestination?.getAction(directions.actionId)?.let { action
```

Comment:
Isn't it a bug if this is called with `directions` for which `getAction` returns `null`? If so we should log an error and ignore.